wordle-web/layouts/default.vue

```diff
@@ -2,7 +2,8 @@
   <v-app>
     <v-app-bar fixed app>
       <router-link to="/" class="text-h4 font-weight-black no-decoration">
-        Wordle
+        <v-icon color="secondary" x-large>mdi-glass-wine</v-icon>
+        Inconceivable!
       </router-link>
       <v-spacer />
       <settings-dialog />
```

Comment:
Hamburger menu on the right ❌
I personally like what you did more, but it's your decision if that specific requirement is crucial or not